showcase/pom.xml

```diff
@@ -31,7 +31,7 @@
 			<dependency>
 				<groupId>org.ocpsoft.prettytime</groupId>
 				<artifactId>prettytime</artifactId>
-				<version>1.0.8.Final</version>
+				<version>5.0.3.Final</version>
```

Comment:
Thanks for doing the update of the showcase apps! Much appreciated. I'm laughing over here at version 1.0.x of PrettyTime 😆